diff --git a/pywikibot/data/api.py b/pywikibot/data/api.py
index ed09a260..17f3c6e7 100644
--- a/pywikibot/data/api.py
+++ b/pywikibot/data/api.py
@@ -1428,7 +1428,7 @@ class Request(MutableMapping):
             'wbcreateclaim', 'wbremoveclaims', 'wbsetclaimvalue',
             'wbsetreference', 'wbremovereferences', 'wbsetclaim',
             'wbcreateredirect',
-            'thank', 'flowthank'
+            'thank', 'flowthank', 'setpagelanguage'
         )
         # Client side verification that the request is being performed
         # by a logged in user, and warn if it isn't a config username.
